docs/configurationProvisioning.md

```diff
@@ -81,7 +81,7 @@ Lightweight M2M port.
 The following request creates the configuration group for devices with type `WeatherBaloon`:
```

Comment:
This PR is related with PR #118? In that case I understand no new entry in CNR is needed.

Reply:
Ok. NTC

Reply:
...although, on the other side, PR is mentioning #113. Should an entry for #113 be added to CNR file?

Reply:
You're right! Someday I will not forget the CNR!

Reply:
Fixed in c9820c8ff5a11f5fe5c697650b695101a6ddf20f